package.json

```diff
@@ -258,6 +268,11 @@
           "command": "mssql.changeDatabase",
           "when": "editorLangId == sql",
           "group": "navigation@4"
+        },
+        {
+          "command": "mssql.executionPlan",
```

Comment:
maybe "mssql.openExecutionPlanFile" will help disambiguate this from future potential execution plan related commands (such as "mssql.toggleActualExecutionPlan").

Reply:
changed it